layouts/layout-basics/src/reset.css

```diff
@@ -0,0 +1,446 @@
+/* http://meyerweb.com/eric/tools/css/reset/
```

Comment:
Could you serve a minimized version of this file (and for `normalize.css`), just for the sake of keeping file sizes small?